src/test/groovy/com/palantir/gradle/docker/DockerRunPluginTests.groovy

```diff
@@ -174,12 +175,12 @@ class DockerRunPluginTests extends AbstractPluginTest {
         BuildResult buildResult = with('docker', 'dockerRemoveContainer', 'dockerRun', 'dockerRunStatus').build()
 
         then:
-        buildResult.task(':dockerRemoveContainer').outcome == TaskOutcome.SUCCESS
+        buildResult.task(':dockerRemoveContainer').outcome == SUCCESS
 
-        buildResult.task(':dockerRun').outcome == TaskOutcome.SUCCESS
+        buildResult.task(':dockerRun').outcome == SUCCESS
         buildResult.output =~ /(?m)HELLO WORLD/
-        buildResult.task(':dockerRunStatus').outcome == TaskOutcome.SUCCESS
-        buildResult.output =~ /(?m):dockerRunStatus\nDocker container 'bar' is STOPPED./
+        buildResult.task(':dockerRunStatus').outcome == SUCCESS
+        buildResult.output =~ /(?m):dockerRunStatus\nDocker container 'foo' is STOPPED./
```

Comment:
No worries, thanks for iterating – keeping style changes/bug fixes separate have generally helped with traceability in the past, so would prefer you save the style changes for another PR if you don’t mind.